modules/serverless-ossm-secret-filtering-net-istio.adoc

```diff
@@ -50,9 +46,4 @@ spec:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
       name: autoscaler
 ----
-<1> Adding this annotation injects an environment variable, `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=true`, to the `net-istio` controller pod.
-+
-[NOTE]
-====
-This annotation is ignored if you set a different value by overriding deployments.
-====
+
```

Comment:
Thank you so much I will try to merge both in one section. The example which we have used of KnativeServing CR for `net-istio`

![image](https://github.com/openshift/openshift-docs/assets/43639538/941d63b5-2c06-4057-aa30-610b519ad63d)

can you provide the same for `net-kourier`? So that we will be having one example each for both and I will include in the doc?